### ISSUE TITLE
DXE-2808 akamai_cloudlets_application_load_balancer does not map description field of /cloudlets/api/v2/origins/{originId}

### ISSUE THREAD

**Issue by aci-payon-ops** [akamai/terraform-provider-akamai#435]
Hi!

I notice that the provider resource `akamai_cloudlets_application_load_balancer` does not map the description field of the `/cloudlets/api/v2/origins/{originId}` API endpoint.
This decription field corresponds to the "Notes" text in the Control Center.
The notes for each balancer are displayed in the "Notes" column in the Load Balancing Manager, which is quite useful.

The `description` key of the resource maps to the version description, not the origin description.
I believe a separate `origin_description` key should be introduced, mapping to the origin description / notes field.

API endpoint documentation:
https://techdocs.akamai.com/cloudlets/v2/reference/put-origin

### Terraform Version
```
Terraform v1.5.1
on linux_amd64
+ provider registry.terraform.io/akamai/akamai v4.1.0
```

### Affected Resource(s)
Please list the resources as a list, for example:
- `akamai_cloudlets_application_load_balancer`

### Expected Behavior
The resource should accept a separate parameter mapping to the description field of the `/cloudlets/api/v2/origins/{originId}` API endpoint.

### Actual Behavior
There is no such parameter available in the resource.
The existing `description` parameter maps to `/cloudlets/api/v2/origins/{originId}/versions` (per-version description/notes).

### Steps to Reproduce
1. Create a load balancer resource.
2. Provision it - you will see the notes field in the control center being empty.
3. Try to set it from Terraform - there is no way to do it.

I hope this can be implemented Thanks!

**Comment by Slonimskaia** [comment]
Hi @aci-payon-ops,

Thank you for your suggestion. We will consider to implement it

Best regards,
Tatiana Slonimskaia

**Comment by lkowalsk-akamai-com** [comment]
This issue should be fixed now with Akamai Terraform Provider 5.5.0 release
